pkg/tnf/handlers/handler_template/handler.tmpl

```diff
@@ -35,17 +35,27 @@ const (
 // adding special variables
 )
 
-// Args function
+// New{{ .UpperHandlername }} returns a new {{ .UpperHandlername }} handler struct.
+// TODO: Add needed parameters to this function and initialize the handler properly.
+func New{{ .UpperHandlername }}(timeout time.Duration) *{{ .UpperHandlername }} {
+	return &{{ .UpperHandlername }}{
+		timeout: timeout,
+		result:  tnf.ERROR,
+		args:    []string{}, // TODO: Add proper execution command.
+	}
+}
+
+// Args returns the initial execution/send command strings for handler {{ .UpperHandlername }}.
 func (h *{{ .UpperHandlername }}) Args() []string {
 	return h.args
 }
 
 // GetIdentifier returns the tnf.Test specific identifier.
 func (h *{{ .UpperHandlername }}) GetIdentifier() {
```

Comment:
can you fix the signature of the function and return a nil. is it allowed ?

Reply:
Done. Surprisingly, the previous "return nil" was successfully compiled.